### PR TITLE
Modernize the resources + other fixes

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,62 @@
...
 
-May work on other platforms in the Debian and RHEL families with or
-without modification.
+- Ubuntu 12.04+
+- RHEL/CentOS, 7.0 and higher only.
+- Fedora
 
-#### Chef
-* Chef 12+
+May work on other platforms in the Debian and RHEL families with or without modification.
```
Comment (josephholsten) [inline README.md]: this is an impressively meaningless line ;-)

File: README.md [hunk 2]
```diff
@@ -82,45 +69,33 @@ snort_service 'snort' do
...
 
-- Author: Joshua Timberman (<joshua@chef.io>)
-- Author: Tim Smith (<tsmith@chef.io>)
-- Copyright 2010-2015, Chef Software, Inc.
+- Author: Joshua Timberman ([joshua@chef.io](mailto:joshua@chef.io))
+- Author: Tim Smith ([tsmith@chef.io](mailto:tsmith@chef.io))
+- Copyright 2015-2016, Cozy Services Ltd.
+- Copyright 2010-2016, Chef Software, Inc.
 
 ```
```
Comment (josephholsten) [inline README.md]: Is there a reason we have the license inline and in LICENSE?

File: attributes/default.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #
-# Cookbook Name:: snort
+# Cookbook:: snort
 # Attributes:: default
 #
-# Copyright 2010-2015, Chef Software, Inc
+# Copyright:: 2010-2016, Chef Software, Inc
```
Comment (josephholsten) [inline attributes/default.rb]: Is `::` a YARD thing?

File: metadata.rb [hunk 4]
```diff
@@ -13,7 +13,7 @@
   supports os
 end
 
-depends 'compat_resource', '>= 12.9.0'
+depends 'compat_resource', '>= 12.14.7'
```
Comment (josephholsten) [inline metadata.rb]: can we say why this version is required?

File: README.md (hunk 1 above)
Comment (tas50) [inline README.md]: Standard meaningless line

File: resources/service_systemd.rb [hunk 5]
```diff
@@ -63,15 +62,14 @@ def create_init
   end
 
   def cleanup_old_service
-    if ::File.exists?('/etc/init.d/snort')
-      service 'disable sys-v init snort' do
-        service_name svc_name
-        action [:stop, :disable]
-      end
+    return unless ::File.exist?('/etc/init.d/snort')
```
Comment (josephholsten) [inline resources/service_systemd.rb]: this is functionally identical, right? just cookstylery?

File: test/integration/default/default_spec.rb [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+if os[:family] == 'centos' || os[:family] == 'fedora'
   describe service('snortd') do
     it { should be_enabled }
     it { should be_running }
   end
 else
   describe service('snort') do
     it { should be_enabled }
-    #it { should be_running }
+    # it { should be_running }
```
Comment (josephholsten) [inline test/integration/default/default_spec.rb]: should this be commented? should it be removed?

File: README.md (hunk 2 above)
Comment (tas50) [inline README.md]: We always have

File: attributes/default.rb (hunk 3 above)
Comment (tas50) [inline attributes/default.rb]: So...I thought it was a yard thing, but it's not. I believe it's an rdoc thing that was adopted for cookbooks, but we never actually did anything about it. I've been setting them in a predictable way so we *could* potentially use them sometime in the future. I suspect what will probably happen is I'll convert this to a more yard like setup at some point.

File: resources/service_systemd.rb (hunk 5 above)
Comment (tas50) [inline resources/service_systemd.rb]: New rubocop thinks you should guard clause anywhere you can.

File: test/integration/default/default_spec.rb (hunk 6 above)
Comment (tas50) [inline test/integration/default/default_spec.rb]: I believe that was working around an InSpec bug that *may* be fixed now

**Pull request merged**